File: EconRentCarWeb/EconRentCar.DataModel/Empleado.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using EconRentCar.Common;
using EconRentCar.Core;

namespace EconRentCar.DataModel
{
    public class Empleado : IEntityBase
    {
        public Empleado()
        {
            Rentas = new HashSet<Renta>();
        }
        public Guid Id { get; set; }
        [Required]
        public string Nombres { get; set; }
        [Required]
        public string Apellidos { get; set; }
        [Required]
        public string CedulaEmpleado { get; set; }
        [Required]
        public TandaLaboral TandaLaboral { get; set; }
        [Required]
        [DataType("money")]
        public decimal PorcentajeComision { get; set; }
        [Required]
        public DateTime FechaIngreso { get; set; }
        public bool Activo { get; set; }
        [ForeignKey("AppUsers")]
        public string AppUserId { get; set; }
        public virtual AppUser AppUsers { get; set; }
        public virtual ICollection<Renta> Rentas { get; set; }

    }
}

File: EconRentCarWeb/EconRentCarClient/src/app/shared/models/empleado.ts
import { BaseData } from '../dynamic-crud/models';
import { TandaLaboral } from './enums';
import { Renta } from './renta';

export class Empleado implements BaseData {
    public Id = '00000000-0000-0000-0000-000000000000';
    public Nombres = '';
    public Apellidos = '';
    public CedulaEmpleado = '';
    public TandaLaboral = TandaLaboral.Matutino;
    public PorcentajeComision = 0.00;
    public FechaIngreso = new Date();
    public AppUserId = '';
    public Rentas: Renta[];
}
